src/MultiLocationEncoder.sol

```diff
@@ -1,37 +1,75 @@
 // SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Also, can you add copyright text please. We'll need to copy over this source code into our repo, and would like to attribute Parity.

Example:
```suggestion
SPDX-FileCopyrightText: 2019 Jane Doe <jane@example.com>
```